### DISCUSSION TITLE
像add_question_sql这种函数需要自己重写吗？有没有大神有代码或者指导一下怎么写

### DISCUSSION THREAD

**Discussion by zyc1128** [vanna-ai/vanna#453] [Q&A]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Error logs/Screenshots**
If applicable, add logs/screenshots to give more information about the issue.

**Desktop (please complete the following information where):**
 - OS: [e.g. Ubuntu]
 - Version: [e.g. 20.04]
 - Python: [3.9]
 - Vanna: [2.8.0]

**Additional context**

**Comment by yushangfu** [comment]
If you use chromadb as your vector store, this function has been implemented in ./chromadb/chromadb_vector.py.
If you use other vector store, you should code this funtion by yourself.
